### ISSUE TITLE
JS code format issue in this documentation  

### ISSUE THREAD

**Issue by asith-w** [MicrosoftDocs/msteams-docs#9857]
There is format error in the **Callbacks converted to promises** [section](https://learn.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/using-teams-client-library?tabs=javascript%2Cmanifest-teams-toolkit#callbacks-converted-to-promises)
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/5784846/1ed4fc0e-2811-4add-ba72-37aee6664081)

[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 4ae8187e-a7e4-4b5a-6f38-da6bfa30f2d2
* Version Independent ID: 4ae8187e-a7e4-4b5a-6f38-da6bfa30f2d2
* Content: [Teams JavaScript client library SDK - Teams](https://learn.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/using-teams-client-library?tabs=javascript%2Cmanifest-teams-toolkit)
* Content Source: [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/main/msteams-platform/tabs/how-to/using-teams-client-library.md)
* Service: **msteams**
* Sub-service: **m365apps**
* GitHub Login: @erikadoyle
* Microsoft Alias: **mosdevdocs**

**Comment by ChetanSharma-msft** [comment]
Hello @asith-w - Thanks for raising your query.
We will look into it and let you know the updates.

**Comment by asith-w** [comment]
> Hello @asith-w - Thanks for raising your query. As per our understanding, you want to remove this part? ![image](https://user-images.githubusercontent.com/59052547/281376462-1a1e706e-dfe3-47d9-967c-a6bccb36440e.png)
> 
> Could you please elaborate more on your ask?

@ChetanSharma-msft I wanted to make sure that I could use 'teams.fullTrust.joinedTeams.getUserJoinedTeams' callbacks. However, before I could ask that question, I needed to determine whether the related code snippet was correct or not. 
  
( it's bit unclear what the documentation is trying to explain. In simpler terms, there is no such function as 'teams.fullTrust.joinedTeams.getUserJoinedTeams' in the Teams JavaScript library.  )

**Comment by Prasad-MSFT** [comment]
@asith-w -  We checked and can confirm that `teams.fullTrust.joinedTeams.getUserJoinedTeams` function is available in Teams JavaScript library. Checked using `@microsoft/teams-js": "^2.5.0`
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/62237649/d72a7385-0671-4bfc-a235-7656adb14f0c)
Definition:
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/62237649/ecdb83f5-117c-4a53-aa21-96f155430dca)

**Comment by ChetanSharma-msft** [comment]
> @asith-w - We checked and can confirm that `teams.fullTrust.joinedTeams.getUserJoinedTeams` function is available in Teams JavaScript library. Checked using `@microsoft/teams-js": "^2.5.0` ![image](https://user-images.githubusercontent.com/62237649/283125070-d72a7385-0671-4bfc-a235-7656adb14f0c.png) Definition: ![image](https://user-images.githubusercontent.com/62237649/283125282-ecdb83f5-117c-4a53-aa21-96f155430dca.png)

Hello @asith-w - Also, wanted to inform you that this method is not available for public use for now:
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/59052547/ddcd0a5d-d2db-4d41-8cf7-e188cd448248)
